meson.build

```diff
@@ -0,0 +1,150 @@
+
+project('openjp2', 'c', 'cpp', version : '2.3.1',
+        meson_version : '>= 0.46.0',
```

Comment:
Please the minimum version that will include at least `0.55.0` or the lowest that includes https://github.com/mesonbuild/meson/pull/6917.